modules/gdscript/gdscript_tokenizer.h

```diff
@@ -169,6 +169,7 @@ class GDScriptTokenizer {
 		int cursor_position = -1;
 		CursorPlace cursor_place = CURSOR_NONE;
 		String source;
+		bool marked_as_indentifier;
```

Comment:
Typo + default value:
```suggestion
		bool marked_as_identifier = false;
```

Reply:
thanks for the typo correction and the advice i corrected the typo at all the places as i was consistent with my typo thanks to autocompletion